test/surface_albedo.jl

```diff
@@ -0,0 +1,201 @@
+using ClimaComms
+using Logging
+using ClimaAtmos
+import Random
+Random.seed!(1234)
+# using Plots
+
+redirect_stderr(IOContext(stderr, :stacktrace_types_limited => Ref(false)))
+
+@testset "set_surface_albedo!" begin
+    config = ClimaAtmos.AtmosConfig()
+    FT = Float32
+
+    # test set_surface_albedo!(Y, p, t, α_type::ConstantAlbedo)
+    config.parsed_args["rad"] = "clearsky"
+    config.parsed_args["FLOAT_TYPE"] = string(FT)
+    config.parsed_args["output_default_diagnostics"] = false
+    simulation = ClimaAtmos.get_simulation(config)
+    (; u, p, t) = simulation.integrator
+    ClimaAtmos.set_surface_albedo!(u, p, t, p.atmos.surface_albedo)
+    @test all(
+        isapprox.(
+            p.radiation.radiation_model.direct_sw_surface_albedo,
+            0.38;
+            atol = eps(FT),
+        ),
+    )
+    @test all(
+        isapprox.(
+            p.radiation.radiation_model.diffuse_sw_surface_albedo,
+            0.38;
+            atol = eps(FT),
+        ),
+    )
```

Comment:
Shouldn't this be exactly 0.38?

Reply:
Oh, actually, I may need to keep it within the eps precision, since `extrema = (0.3799999952316284, 0.3799999952316284)`

Reply:
Do you get exactly equal if you add FT before 0.38?

Reply:
I tried this:
```
julia> a = Float32(0.38)
0.38f0

julia> a == 0.38
false

julia> a == Float32(0.38)
true
```

Reply:
Ahh yeah, well spotted. This works too!